cql3/query_processor.cc

```diff
@@ -250,12 +327,12 @@ query_processor::query_processor(service::storage_proxy& proxy, database& db, qu
                     sm::make_counter(
                             "reverse_queries",
                             _cql_stats.reverse_queries,
-                            sm::description("Counts number of CQL SELECT requests with ORDER BY DESC.")),
+                            sm::description("Counts the number of CQL SELECT requests with ORDER BY DESC.")),
```

Comment:
The description is not correct, but I understand you only fixing the grammar.

those are requests with reverse ORDER BY.